### PR TITLE
Calculate estimated APR for a specific price range

### PR DESCRIPTION
**Pull request by rawfalafel**

### CODE REVIEW

File: src/utils/public/index.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
-export * from "./percentage";
-export * from "./tick";
+export { TokenUSDPrices } from "../token-price";
```
Comment (philcchen) [inline src/utils/public/index.ts:1]: wasn't sure what the semantics were behind putting things in public or not?

File: src/utils/public/apr.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+export const ZERO_APR = {
+  fee: 0,
+  rewards: [0, 0, 0],
+};
+
+export function estimateAprsForPriceRange(
+  pool: PoolData,
+  // TODO: should this actually be fetched/shared?
+  // There's a weird in/out/in dependency here on prices/fees
```
Comment (rawfalafel) [inline src/utils/public/apr.ts:23]: Hmm, so it looks like [getTokenUSDPrices](https://github.com/orca-so/whirlpool-sdk/blob/main/src/utils/token-price.ts#L23) caches the only async calls that it makes, so i think it's pretty reasonable to make the call in here as well.

File: src/utils/public/index.ts (hunk 1 above)
Comment (rawfalafel) [reply src/utils/public/index.ts:1]: i think @odcheung will do a pass on stuff like this when he refactors the SDK

File: src/utils/public/apr.ts (hunk 2 above)
Comment (philcchen) [reply src/utils/public/apr.ts:23]: This only takes into account the whirlpools and not standard pools; for rewards where the token is not listed in whirlpools directly, the price cannot be derived from this call.

**Comment by rawfalafel** [comment]
Yea ok, then we should ship as is and include a comment

**Pull request merged**